resources/populateDynamoDB.ts

```diff
@@ -0,0 +1,65 @@
+
+import { DynamoDB } from 'aws-sdk';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+// Read mock data from JSON files
+const products = JSON.parse(readFileSync(join(__dirname, '..', 'data', 'products.json'), 'utf-8'));
+const categories = JSON.parse(readFileSync(join(__dirname, '..', 'data', 'categories.json'), 'utf-8'));
+
+
+const dynamoDB = new DynamoDB.DocumentClient({
+  region: 'us-east-1'
+});
+// DynamoDB table name
+const TABLE_NAME = 'OnlineShopStack-OnlineShopTable008A5D70-1XU04MP3BSOZ4';
```

Comment:
Introducing a `.env` with all these credentials/configs probably would have been a better way to handle this.

Reply:
completed